src/sage/stats/distributions/discrete_gaussian_lattice.py

```diff
@@ -1,30 +1,31 @@
 # -*- coding: utf-8 -*-
-r"""
-Discrete Gaussian Samplers over Lattices
+r""" Discrete Gaussian Samplers over Lattices
```

Comment:
```suggestion
r"""
Discrete Gaussian Samplers over Lattices
```
This is how we do our documentation. All one-lines begin on the next line. (Also, IMO, it is also easier to parse the intent and read it.)